packages/manager/apps/zimbra/src/pages/dashboard/MailingLists/MailingLists.tsx

```diff
@@ -0,0 +1,203 @@
+import ActionButtonMailingList from './ActionButtonMailingList';
+import LabelChip from '@/components/LabelChip';
+import { IAM_ACTIONS } from '@/utils/iamAction.constants';
+import { ResourceStatus } from '@/api/api.type';
+import { MailingListType } from '@/api/mailinglist';
+
+export type MailingListItem = {
+  id: string;
+  name: string;
+  organizationLabel: string;
+  organizationId: string;
+  owner: string;
+  aliases: number;
+  moderators: number;
+  subscribers: number;
+  status: ResourceStatus;
+};
+
+const columns: DatagridColumn<MailingListItem>[] = [
+  {
+    id: 'domains',
+    cell: (item) => (
+      <OsdsText
+        color={ODS_THEME_COLOR_INTENT.text}
+        size={ODS_TEXT_SIZE._100}
+        level={ODS_TEXT_LEVEL.body}
+      >
+        {item.name}
+      </OsdsText>
+    ),
+    label: 'zimbra_mailinglists_datagrid_name_label',
+  },
+  {
+    id: 'organization',
+    cell: (item) =>
+      item.organizationLabel && (
+        <LabelChip id={item.organizationId}>{item.organizationLabel}</LabelChip>
+      ),
+    label: 'zimbra_mailinglists_datagrid_organization_label',
+  },
+  {
+    id: 'owner',
+    cell: (item) => (
+      <OsdsText
+        color={ODS_THEME_COLOR_INTENT.text}
+        size={ODS_TEXT_SIZE._100}
+        level={ODS_TEXT_LEVEL.body}
+      >
+        {item.owner}
+      </OsdsText>
+    ),
+    label: 'zimbra_mailinglists_datagrid_owner_label',
+  },
+  {
+    id: 'aliases',
+    cell: (item) => (
+      <OsdsText
+        color={ODS_THEME_COLOR_INTENT.text}
+        size={ODS_TEXT_SIZE._100}
+        level={ODS_TEXT_LEVEL.body}
+      >
+        {item.aliases}
+      </OsdsText>
+    ),
+    label: 'zimbra_mailinglists_datagrid_aliases_label',
+  },
+  {
+    id: 'moderators',
+    cell: (item) => (
+      <OsdsText
+        color={ODS_THEME_COLOR_INTENT.text}
+        size={ODS_TEXT_SIZE._100}
+        level={ODS_TEXT_LEVEL.body}
+      >
+        {item.moderators}
+      </OsdsText>
+    ),
+    label: 'zimbra_mailinglists_datagrid_moderators_label',
+  },
+  {
+    id: 'subscribers',
+    cell: (item) => (
+      <OsdsText
+        color={ODS_THEME_COLOR_INTENT.text}
+        size={ODS_TEXT_SIZE._100}
+        level={ODS_TEXT_LEVEL.body}
+      >
+        {item.subscribers}
+      </OsdsText>
+    ),
+    label: 'zimbra_mailinglists_datagrid_subscribers_label',
+  },
+  {
+    id: 'tooltip',
+    cell: (item) =>
+      (item.status === ResourceStatus.READY ||
+        item.status === ResourceStatus.ERROR) && (
+        <ActionButtonMailingList mailingListItem={item} />
+      ),
+    label: '',
+  },
+];
+
+export const getMailingListItem = (data: MailingListType): MailingListItem => {
+  return {
+    id: data?.id,
+    name: data?.currentState?.email,
+    organizationLabel: data?.currentState?.organizationLabel,
+    organizationId: data?.currentState?.organizationId,
+    owner: data?.currentState?.owner?.currentState?.email || 'test@owner.com',
```

Comment:
```suggestion
    owner: data?.currentState?.owner?.currentState?.email,
```
Why we need hardcoded value ?

Reply:
for now the api sends an id as the "owner" property, I'm not sure if it will change or not, if I need to fetch the user my self or not.. So it was for testing purposes

Reply:
Ok added your suggestion